x-pack/plugins/security/server/plugin.ts

```diff
@@ -74,6 +76,7 @@ export interface PluginSetupDependencies {
   features: FeaturesPluginSetup;
   licensing: LicensingPluginSetup;
   taskManager: TaskManagerSetupContract;
+  usageCollection: UsageCollectionSetup;
```

Comment:
NIT: `usageCollection` is in the list of `optionalPlugins`. Should it be marked as optional in the types as well?

Reply:
Great catch, thanks!